### DISCUSSION TITLE
Relation Extractor Args

### DISCUSSION THREAD

**Discussion by darrkj** [explosion/spaCy#11398] [Help: Coding & Implementations]
I have trained a custom relation extractor component and have that working in my pipeline but to expand on the model I am having some issues. The main issue is that I need to augment parts of the annotation process to get observations for training. I can see in the prodigy page that you can add entity labels, via `--add-ents` to the rel.manual recipe [here](https://prodi.gy/docs/recipes#rel-manual). However other aspects require this to be a custom recipe. The issue is that I cant find any content on how arguments like '--add-ents` or `--add-nps` would exist in the custom recipe. Is there a way they can be added to the custom recipe in some way or does that change require implementing and can no longer utilize the command line args.

**Comment by polm** [comment]
Command line switches like `--add-ents` will preprocess the incoming data and should work the same whether you're using custom recipes or not. 

Note that questions specifically about Prodigy like this are usually better asked at the [Prodigy forums](https://support.prodi.gy/).
